.semaphore/semaphore.yml

```diff
@@ -3,7 +3,8 @@ name: Apache Druid
 agent:
   machine:
     type: s1-prod-ubuntu20-04-amd64-1
-
+execution_time_limit:
+  hours: 3
```

Comment:
did execution time go up after we removed/changed the caching behavior?

Reply:
The only changes to the base branch of `24.0.2-confluent` are in this PR so it looks to have gone down by about 3 minutes:

`24.0.2-confluent` is `40:15` in duration: https://semaphore.ci.confluent.io/workflows/8b25f795-d57b-4315-8ff9-2dfb395580c4?pipeline_id=76afc9e8-0e68-45fd-acbf-258c8ab29285
This PR build is `36:59` in duration: https://semaphore.ci.confluent.io/workflows/fd3ebfac-2f5a-4b5f-8f4c-be497427b83b?pipeline_id=216c972e-b9b3-4b69-ab60-d460ffeddd28

Reply:
ah ok, do we need this override then?

Reply:
removed project level cache to correct for some assumptions. resolving.